phangsPipeline/casaVisRoutines.py

```diff
@@ -1413,70 +1479,83 @@ def build_mstransform_call(
 
         if vstart_kms is None:
             logger.error("Please specify a starting velocity in km/s.")
-            raise Exception("No starting velocity specified.")        
-        start_vel_string =  ("{:12.8f}".format(vstart_kms)+'km/s').strip()
-        
+            raise Exception("No starting velocity specified.")
+        start_vel_string = ("{:12.8f}".format(vstart_kms)+'km/s').strip()
+
         # Check that we have a velocity width
 
         if vwidth_kms is None:
             if nchan is None:
-                logger.error("Please specify a velocity width in km/s or number of channels.")
+                logger.error(
+                    "Please specify a velocity width in km/s or "
+                    "number of channels.")
                 raise Exception("No velocity width specified.")
```

Comment:
Copy the logger message into the exception for clarity.

Reply:
Done